src/modules/accessory.json

```diff
@@ -0,0 +1,202 @@
+          "schema": {
+            "type": "string"
+          }
+        },
+        {
+            "name": "connectivity",
+            "summary": "Defines the mechanism by which accessory will connect to the device.",
+            "schema": {
+              "type": "string"
+            }
+        },
+        {
+          "name": "timeoutInSecs",
+          "summary": "Defines the timeout in seconds. If the threshold for timeout is passed without a result it will throw an error.",
+          "schema": {
+            "type": "integer"
+          }
+        }
+      ],
+      "result": {
+        "name": "pairedAccessory",
+        "summary": "The state of last paired accessory",
+        "schema": {
+          "$ref": "#/components/schema/AccessoryInfo"
+        }
+      },
+      "examples": [
+        {
+          "name": "Pair a Bluetooth Remote",
+          "params": [
+            {
+              "name": "type",
+              "value": "remote"
+            },
+            {
+              "name": "connectivity",
+              "value": "bluetooth"
+            },
+            {
+              "name": "timeoutInSecs",
+              "value": 180
+            }
+          ],
+          "result": {
+            "name": "Bluetooth Remote successful pairing example",
+            "value": {
+              "type": "remote",
+              "make": "UEI",
+              "model": "PR1",
+              "connectivity": "bluetooth"
+            }
+          }
+        },
+        {
+          "name": "Pair a Bluetooth Speaker",
+          "params": [
+            {
+              "name": "type",
+              "value": "speaker"
+            },
+            {
+                "name": "connectivity",
+                "value": "bluetooth"
+            },
+            {
+              "name": "timeoutInSecs",
+              "value": 180
+            }
+          ],
+          "result": {
+            "name": "Bluetooth Speaker successful pairing example",
+            "value": {
+              "type": "speaker",
+              "make": "Sonos",
+              "model": "V120",
+              "connectivity": "bluetooth"
+            }
+          }
+        },
+        {
+          "name": "Pair a RF Remote",
+          "params": [
+            {
+              "name": "type",
+              "value": "remote"
+            },
+            {
+                "name": "connectivity",
+                "value": "rf"
+            },
+            {
+              "name": "timeoutInSecs",
+              "value": 180
+            }
+          ],
+          "result": {
+            "name": "RF Remote successful pairing example",
+            "value": {
+              "type": "remote",
+              "make": "UEI",
+              "model": "15",
+              "connectivity": "rf"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "list",
+      "summary": "Provides a list of Accesories paired to the device.",
+      "params": [],
```

Comment:
Probably worth providing optional type and connectivity params to filter the list.

Reply:
Done and also added examples to help us with Ref UI